FilePersistence/src/version_control/GitPiecewiseLoader.cpp

```diff
@@ -106,8 +114,8 @@ NodeData GitPiecewiseLoader::parseGrepLine(const QString& line, bool isWorkDir)
 	// first element is revision
 	else
 	{
-		nodeData.persistentUnit_ = line.section(':', 1, 1);
-		nodeData.nodeLine_ = line.section(':', 2);
+		nodeData.persistentUnit_ = line.section(':', 0, 0);
```

Comment:
actually, if you look at the two branches of this if statement, you will see that now both branches are identical after our changes. Please simplify this whole method by removing the `if`